.code-samples.meilisearch.yaml

```diff
@@ -278,6 +278,14 @@ search_parameter_guide_crop_marker_1: |-
     'attributesToCrop': ['overview'],
     'cropMarker': '[…]'
   })
+search_parameter_guide_matching_strategy_1: |-
+  client.index('movies').search('big fat liar', { 
+    matchingStrategy: 'last'
+  })
+search_parameter_guide_matching_strategy_2: |-
+  client.index('movies').search('big fat liar', { 
+    matchingStrategy: 'all'
```

Comment:
```suggestion
    'matchingStrategy': 'all'
```

Reply:
Same and can you check the extra space too?